### ISSUE TITLE
capital letter in username prompt on mobile

### ISSUE THREAD

**Issue by ita1024** [diaspora/diaspora#5955]
The username prompt to login on firefox/desktop does not capitalize the first letter. Yet on android (firefox) or iphone (safari), the first letter is capitalized :collision:

The desktop site *correctly* styles the element with:
  autocapitalize="none" autocorrect="off" autofocus="autofocus"
Yet, the mobile version styles the element with only:
  autofocus="autofocus"

Diaspora 0.5.0.1

**Comment by svbergerem** [comment]
https://github.com/diaspora/diaspora/blob/develop/app/views/sessions/new.mobile.haml#L30

**Comment by ita1024** [comment]
This seems to work for me.

```diff
diff --git a/app/views/sessions/new.mobile.haml b/app/views/sessions/new.mobile.haml
index 02a6803..8cd09f3 100644
--- a/app/views/sessions/new.mobile.haml
+++ b/app/views/sessions/new.mobile.haml
@@ -27,7 +27,7 @@
             .control-group
               = f.label :username, t('username')
               .controls
-                = f.text_field :username, :autofocus => true
+                = f.text_field :username, :autofocus => true, :autocapitalize => 'none', :autocorrect => 'off'
 
             .control-group
               = f.label :password , t('password')
```

**Comment by svbergerem** [comment]
@ita1024 Would you mind opening a pull request?

**Comment by ita1024** [comment]
https://github.com/diaspora/diaspora/pull/5961

**Comment by svbergerem** [comment]
#5961 would have fixed this issue. There were just some small changes needed because of our ruby styleguide but unfortunately @ita1024 closed the pull request.

To fix this issue just look at the changes in #5961 and my comment over there.
